src/MockBot.sln

```diff
@@ -7,6 +7,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MockBot.UnitTests", "MockBo
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MockBot.Api", "MockBot.Api\MockBot.Api.csproj", "{D0ED79F9-0883-41AF-93BC-E4BA9C0EA29A}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{9D2E6422-BEED-4DB2-890E-05DCD64B3B9B}"
```

Comment:
This is an interesting approach. Does this apply a single `.editorconfig` to all projects in the solution and does it work? If so, we should incorporate this approach into this PR: https://github.com/buerokratt/Cross-functional-requirements/pull/40

Reply:
This one I do not know about, another auto generated change. Whatever you say I should do, goes.

Reply:
In the other repos we have an `.editorconfig` in each project (i.e. `*.csproj` level, multiple projects per solution). I suggest we do the same here for consistency. See the guide at https://github.com/buerokratt/Cross-functional-requirements/blob/main/docs/technical/code-analysis.md

Reply:
Alright, will look into that and try to bring my stuff in line

Reply:
Moved .editorconfig files now